src/config/config.js

```diff
@@ -6,6 +6,17 @@ export const LOGIN_SERVER_ADDRESS = {
   loginSignUpAddress: `${loginServerRootAddress}/users/signup`,
 };
 
+// TODO: 각자 테스트한 후, 최종적으로 아래와 같이 변경;
+// const BASE_URL = 'http://10.57.2.129:8000';
+// const API = {
+//   loginMainAddress: `${BASE_URL}/users/check`,
+//   loginSignInAddress: `${BASE_URL}/users/login`,
+//   loginSignUpAddress: `${BASE_URL}/users/signup`,
+//   category: `${BASE_URL}/products/categories`,
+//   allProducts: `${BASE_URL}/products`,
```

Comment:
```suggestion
//   category: `${BASE_URL}/products/categories/`,
//   allProducts: `${BASE_URL}/products/`,
```
마지막에`/`이게 없어서 사용할때마다 `/`를 붙여줘야했습니다.

Reply:
멘붕 온 와중에 위에 댓글 넘나따뜻..
URL 수정완료하였습니다.